packages/web/components/Patron/PatronTile.tsx

```diff
@@ -82,9 +83,16 @@ export const PatronTile: React.FC<Props> = ({ index, patron }) => {
                 {patron.pSeedBalance != null && (
                   <WrapItem>
                     <MetaTag size="md">
-                      {`pSEED: ${Math.floor(
-                        Number(utils.formatEther(patron.pSeedBalance)),
-                      )}`}
+                      {`$${(
+                        parseFloat(
+                          utils.formatUnits(
+                            patron.pSeedBalance,
+                            Constants.PSEED_DECIMALS,
+                          ),
+                        ) * pSeedPrice
+                      ).toLocaleString(undefined, {
+                        maximumFractionDigits: 0,
+                      })}`}
```

Comment:
I'd put this after `const description`:

```
  const pSeedAmt = parseFloat(utils.formatUnits(
    patron.pSeedBalance,
    Constants.PSEED_DECIMALS,
  ))
  const pSeedBalance = `${Math.floor(pSeedAmt).toLocaleString()} pSEED`
  const displayBalance = (
    pSeedPrice == null ? (
      pSeedBalance
    ) : (
      `$${(
        pSeedAmt * pSeedPrice
      ).toLocaleString(undefined, {
        maximumFractionDigits: 0,
      })}`
  )}
```

```suggestion
                    <Tooltip label={pSeedBalance}>
                      <MetaTag size="md">{displayBalance}</MetaTag>
                    </Tooltip>
```